src/test/java/com/spotify/sparkey/extra/ThreadLocalWithFallbackTest.java

```diff
@@ -0,0 +1,77 @@
+package com.spotify.sparkey.extra;
+
+import org.junit.Test;
+
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ThreadLocalWithFallbackTest {
```

Comment:
missing a test when you try to set initial value to a `null` value

Reply:
Right, will think about what is the correct behavior in that case